internal/webhook/v1/aerospikecluster_mutating_webhook.go

```diff
@@ -714,8 +714,8 @@ func setDefaultsInConfigMap(
 ) error {
 	for k, v := range defaultConfigs {
 		// Special handling.
-		// Older baseValues are parsed to int64 but defaults are in int
-		if newV, ok := v.(int); ok {
+		// Older baseValues are parsed to int64 but defaults are in int32
+		if newV, ok := v.(int32); ok {
```

Comment:
Check this change once against rack-id set flow in mutation webhook.
Mutations sets rack-id as int not int32